### ISSUE TITLE
Expand data dictionary

### ISSUE THREAD

**Issue by peterjmartinson** [belcurv/whobot#24]
Since release, people have tried to add a bunch of new skills.  Many of these should be normalized in the Data Dictionary.

**Comment by peterjmartinson** [comment]
Here is a list of candidates to add:

.NET: 2
anguler 2: 1
apache: 1
ASP: 1
backend: 1
back-end: 1
closure: 1
closures: 1
design: 1
Design: 1
ejs: 1
GitHub: 1
Github: 1
Grommet.js: 1
Heroku: 2
javscript: 1
jekyll: 1
keras: 1
machine learning: 1
MachineLearning: 1
Math: 1
math: 2
maths: 1
mongoose: 1
Mongoose: 4
objective-c: 1
phaser: 1
promises: 2
pug: 1
react-native: 2
react-router: 3
Regex: 1
Regular Expressions: 1
scss: 1
SCSS: 2
socketio: 1
spring: 1
surge: 1
tensorflow: 1
this: 2
webpack 2: 1
webpack: 3

**Comment by peterjmartinson** [comment]
The dictionary has been expanded, and the database has been update.
